### ISSUE TITLE
Interrupt evaluating support?

### ISSUE THREAD

**Issue by fakeGenuis** [WolframResearch/WolframLanguageForJupyter#144]
When evaluating takes a long time, are there some method to interrupt calculation but remain kernel connected?

For example, evaluate `Integrate[1/Sqrt[x^4 + a*x^2 + b], x]` and click interrupt immediately, got popup
![feh_stdin_jdmVxJ](https://user-images.githubusercontent.com/32349731/217168127-199e0169-52d3-4fdd-a0ff-7aa44d457f76.png)
neither click yes or no, the kernel exit and need restart to reconnect. Test in wolfram mathematica 13.2.

**Comment by fakeGenuis** [comment]
Hi guys, I write a similar project that support interrupt, by receiving message from jupyter frontend. The evaluation is done inside an external link, so interrupt not lose connection to kernel manage it. Now it is poorly featured, if you are interested, see https://github.com/fakeGenuis/wolframForJupyter.git

**Comment by RunLiz** [comment]
Wow，cool 
not fakeGenuis 
TrueGenuis!!!!!!!

**Comment by cc-wr** [comment]
Looks cool!

Just generally, I will say that I am happy to look at pull requests that improve this project as well.

**Comment by fakeGenuis** [comment]
> Looks cool!
> 
> Just generally, I will say that I am happy to look at pull requests that improve this project as well.

Yeah, I really want to contribute upon this project at first. But the complexity of it prevent me from that，especially the evaluation and request handler parts. Many features are still missing in the new project, which makes it more simple to construct and debug with just one addition interrupt support. Anyway, this project are really well structured and I borrow many implements from it.
